<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>639</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>18</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleLabel" name="TitleLabel">
     <property name="text">
      <string>设置页面 q(≧▽≦q)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SmoothScrollArea" name="SmoothScrollArea">
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>557</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="CaptionLabel" name="CaptionLabel_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>设置显示方式与内容</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="lightColor" stdset="0">
            <color alpha="150">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </property>
           <property name="darkColor" stdset="0">
            <color alpha="200">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_5">
                <property name="text">
                 <string>启用明日课程提醒</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>在指定时间弹出提示，显示明天课程相关信息</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="SwitchButton" name="enableTip">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_6">
                <property name="text">
                 <string>显示课程数量</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>设置提醒中显示的明日课程数量</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="SpinBox" name="SpinBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_9">
                <property name="text">
                 <string>通知显示时间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>设置提醒的显示时间（秒）</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="SpinBox" name="SpinBox_2"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_8">
                <property name="text">
                 <string>显示发送时间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>设置发送通知的指定时间 (tip:不应与其他通知过近)</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="TimePicker" name="TimePicker">
              <property name="maximumSize">
               <size>
                <width>240</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="timeLayout">
              <property name="spacing">
               <number>8</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="StrongBodyLabel" name="StrongBodyLabel_7">
                <property name="text">
                 <string>不显示的课程</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CaptionLabel" name="CaptionLabel_4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>设置不希望在提醒中显示的课程名称（多个课程用逗号分隔）</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="lightColor" stdset="0">
                 <color alpha="150">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </property>
                <property name="darkColor" stdset="0">
                 <color alpha="200">
                  <red>255</red>
                  <green>255</green>
                  <blue>255</blue>
                 </color>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="LineEdit" name="excludedCoursesEdit">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>30</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>如：体育,音乐,美术</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="CardWidget" name="CardWidget_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="leftMargin">
             <number>16</number>
            </property>
            <property name="topMargin">
             <number>16</number>
            </property>
            <property name="rightMargin">
             <number>16</number>
            </property>
            <property name="bottomMargin">
             <number>16</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>160</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="PrimaryPushButton" name="PrimaryPushButton">
              <property name="text">
               <string>测试通知</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>160</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimePicker</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
